installers/rpm-code/dev-build.sh

```diff
@@ -14,9 +14,9 @@ fi
 if [[ $1 == common ]]; then
     cd ~/src/radiasoft/container-rpm-code
     if [[ ! $(docker images | grep radiasoft/fedora) ]]; then
-        cd container-fedora
-        # in case set by dev-env.sh, because server isn't running yet
+        cd ../container-fedora
         radia_run container-build
+        cd -
```

Comment:
Can you explain the change to this file?

Reply:
It is an unrelated bug I saw. The `cd container-fedora` was relative to `cd ~/src/radiasoft/container-rpm-code`, and `container-fedora` is at the same level, and alternative coding would be:

```bash
    if [[ ! $(docker images | grep radiasoft/fedora) ]]; then
        cd ~/src/radiasoft/container-fedora
        radia_run container-build
    fi
    cd ~/src/radiasoft/container-rpm-code
    radia_run container-build
```

However, that denormalizes `~/src/radiasoft/` so it's a tiny bit better to do it the other way.